projects/plugins/jetpack/tests/runner.js

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env node
 require.extensions[ '.svg' ] = () => 1;
 
-require( '@babel/register' );
+require( '@babel/register' )( {
+	ignore: [ /node_modules/ ],
+	only: [ '.', '../../js-packages/' ],
```

Comment:
Same comment as projects/plugins/jetpack/babel.config.js?

Reply:
We still need this, Yarn link doesn't solve this problem.
The `js-packages` directory is located outside the Babel root dir, so the package doesn't get built when tests run.
I know it's hacky, but gets the job donw.

This is temporary anyway, because once we move the Jetpack testing script into the repo root, `js-packages` will be inside the Babel root dir, and will get built by default.